<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigurationView</class>
 <widget class="QWidget" name="ConfigurationView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,5,1,1">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,4,1">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Network File</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="networkFile"/>
       </item>
       <item>
        <widget class="QPushButton" name="importNetwork">
         <property name="text">
          <string>Import...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,4,1">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Input File</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="inputFile"/>
       </item>
       <item>
        <widget class="QPushButton" name="chooseInput">
         <property name="text">
          <string>Import...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,1">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5" stretch="1,1,1">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,1">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Robustness Type</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="robustnessType">
             <item>
              <property name="text">
               <string>LINFBALL</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>BRIGHTNESS</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>CUSTOMBOX</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,2,1">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Disturbance</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="deltaSlider">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="deltaSpin"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="run">
           <property name="text">
            <string>Run</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
